posts/inside-rust/2024-09-06-leadership-council-update.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "September 2024 Leadership Council Update"
+author: Eric Huss
+team: Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>
+---
+
+Hello again from the Rust Leadership Council!
+We wanted to share an update on what the Council has been working on since [our last update][update].
+
+[update]: https://blog.rust-lang.org/inside-rust/2024/05/14/leadership-council-update.html
+
+## Accomplishments so far
+
+### Welcome James Munns to the Council
+
+As [announced in May](https://blog.rust-lang.org/inside-rust/2024/05/28/launching-pad-representative.html), James Munns joined the Council representing the Launching Pad.
+
+### All-hands and 10-year anniversary
+
+We have been coordinating with [RustNL](https://rustnl.org/) for hosting an all-hands event for all Rust project members. We will also be celebrating Rust's 10-year anniversary at RustWeek 2025.
+
+Part of this coordination has involved ensuring we are capable of handling the travel expenses, and reserving funds for the event itself if needed.
+
+### RustConf 2024
+
+We have continued to coordinate with planning for [RustConf](https://rustconf.com/). James Munns and Jack Huey will be giving a keynote talk called "An Outsider's Guide to the Rust Project", which will focus on the teams and structure that make up the Rust Project today, how the teams communicate and make collaborative decisions, and how new people and groups can get involved.
+
+### Ubuntu Summit 2024
+
+We coordinated with [Ubuntu Summit](https://events.canonical.com/event/51/) for representing the Rust project. Eric Holk has volunteered to give a talk at the event.
+
+### Leadership Council survey
+
+We performed a survey for Rust project members to provide feedback on the effectiveness of the Council. A summary of the results was published at <https://blog.rust-lang.org/2024/08/26/council-survey.html>.
+
+### Foundation fellowship grants
+
+We coordinated with the Foundation for reviewing applications for the [2024 Fellowship Program](https://foundation.rust-lang.org/grants/fellowships/). Information should currently be going out to the awardees, and more will be announced soon.
+
+### Additional items
+
+And a few other items:
+
+* Coordinated with ACM SIGPLAN for receiving the [Programming Languages Software Award](https://www.sigplan.org/Awards/Software/).
+* Approved to have Rust included in [Discord's 10-year anniversary post](https://blog.discourse.org/2024/08/celebrating-a-decade-of-discourse/).
```

Comment:
```suggestion
* Approved to have Rust included in [Discourse's 10-year anniversary post](https://blog.discourse.org/2024/08/celebrating-a-decade-of-discourse/).
```

From what I can tell these are two separate projects?

Reply:
Oh my! Thanks for the correction!